src/discof/restore/fd_snaprd_tile.c

```diff
@@ -510,6 +551,16 @@ after_credit( fd_snaprd_tile_t *  ctx,
   (void)charge_busy;
 
   long now = fd_log_wallclock();
+
+  /* If snapshots are read from disk, we can immediatley publish an
+     update notificaiton with the snapshot slot and load path */
+  if( FD_UNLIKELY( ctx->local_in.full_snapshot_slot!=ULONG_MAX && !ctx->gui_incremental_path_published ) ) {
+    snapshot_path_gui_publish( ctx, stem, ctx->local_in.full_snapshot_path, /* is_full */ 1 );
```

Comment:
```suggestion
    snapshot_path_gui_publish( ctx, stem, ctx->local_in.full_snapshot_path, /* is_full */ 1 );
    *opt_poll_in = 0;
    return;
```

I think do this otherwise you have to consider what the new value of STEM_BURST should be